Upstream-Status: Undecided

This patch removes some of hardcoded references to /lib
and /usr/lib since on some architectures it should be
/lib64 and /usr/lib64 atleast in OE

I am not sure about the intention of hardcoded values
thats why status is undecided

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: git/Makefile.am
===================================================================
--- git.orig/Makefile.am	2012-09-22 11:07:58.811981199 -0700
+++ git/Makefile.am	2012-09-22 11:09:11.267983956 -0700
@@ -64,25 +64,25 @@
 
 # Our own, non-special dirs
 pkgsysconfdir=$(sysconfdir)/systemd
-userunitdir=$(prefix)/lib/systemd/user
-userpresetdir=$(prefix)/lib/systemd/user-preset
-tmpfilesdir=$(prefix)/lib/tmpfiles.d
-sysctldir=$(prefix)/lib/sysctl.d
-usergeneratordir=$(prefix)/lib/systemd/user-generators
+userunitdir=$(prefix)/$(rootlibdir)/systemd/user
+userpresetdir=$(prefix)/$(rootlibdir)/systemd/user-preset
+tmpfilesdir=$(prefix)/$(rootlibdir)/tmpfiles.d
+sysctldir=$(prefix)/$(rootlibdir)/sysctl.d
+usergeneratordir=$(prefix)/$(rootlibdir)/systemd/user-generators
 pkgincludedir=$(includedir)/systemd
 systemgeneratordir=$(rootlibexecdir)/system-generators
 systemshutdowndir=$(rootlibexecdir)/system-shutdown
 systemsleepdir=$(rootlibexecdir)/system-sleep
-systemunitdir=$(rootprefix)/lib/systemd/system
-systempresetdir=$(rootprefix)/lib/systemd/system-preset
-udevlibexecdir=$(rootprefix)/lib/udev
+systemunitdir=$(rootprefix)/$(rootlibdir)/systemd/system
+systempresetdir=$(rootprefix)/$(rootlibdir)/systemd/system-preset
+udevlibexecdir=$(rootprefix)/$(rootlibdir)/udev
 udevhomedir = $(udevlibexecdir)
 udevrulesdir = $(udevlibexecdir)/rules.d
 
 # And these are the special ones for /
 rootprefix=@rootprefix@
 rootbindir=$(rootprefix)/bin
-rootlibexecdir=$(rootprefix)/lib/systemd
+rootlibexecdir=$(rootprefix)/$(rootlibdir)/systemd
 
 CLEANFILES = $(BUILT_SOURCES)
 EXTRA_DIST =
@@ -132,7 +132,7 @@
 	-DSYSTEMD_STDIO_BRIDGE_BINARY_PATH=\"$(bindir)/systemd-stdio-bridge\" \
 	-DROOTPREFIX=\"$(rootprefix)\" \
 	-DRUNTIME_DIR=\"/run\" \
-	-DRANDOM_SEED=\"$(localstatedir)/lib/random-seed\" \
+	-DRANDOM_SEED=\"$(localstatedir)/$(rootlibdir)/random-seed\" \
 	-DSYSTEMD_CRYPTSETUP_PATH=\"$(rootlibexecdir)/systemd-cryptsetup\" \
 	-DSYSTEM_GENERATOR_PATH=\"$(systemgeneratordir)\" \
 	-DUSER_GENERATOR_PATH=\"$(usergeneratordir)\" \
@@ -2692,7 +2692,7 @@
 
 binfmt-install-data-hook:
 	$(MKDIR_P) -m 0755 \
-		$(DESTDIR)$(prefix)/lib/binfmt.d \
+		$(DESTDIR)$(prefix)/$(rootlibdir)/binfmt.d \
 		$(DESTDIR)$(sysconfdir)/binfmt.d \
 		$(DESTDIR)$(systemunitdir)/sysinit.target.wants
 	( cd $(DESTDIR)$(systemunitdir)/sysinit.target.wants && \
@@ -3107,7 +3107,7 @@
 
 timedated-install-data-hook:
 	$(MKDIR_P) -m 0755 \
-		$(DESTDIR)$(prefix)/lib/systemd/ntp-units.d \
+		$(DESTDIR)$(prefix)/$(rootlibdir)/systemd/ntp-units.d \
 		$(DESTDIR)$(sysconfdir)/systemd/ntp-units.d
 	( cd $(DESTDIR)$(systemunitdir) && \
 		rm -f dbus-org.freedesktop.timedate1.service  && \
@@ -3337,7 +3337,7 @@
 logind-install-data-hook:
 	$(MKDIR_P) -m 0755 \
 		$(DESTDIR)$(systemunitdir)/multi-user.target.wants \
-		$(DESTDIR)$(localstatedir)/lib/systemd
+		$(DESTDIR)$(localstatedir)/$(rootlibdir)/systemd
 	( cd $(DESTDIR)$(systemunitdir) && \
 		rm -f dbus-org.freedesktop.login1.service && \
 		$(LN_S) systemd-logind.service dbus-org.freedesktop.login1.service)
@@ -3494,7 +3494,7 @@
 		-e 's,@PACKAGE_VERSION\@,$(PACKAGE_VERSION),g' \
 		-e 's,@PACKAGE_NAME\@,$(PACKAGE_NAME),g' \
 		-e 's,@PACKAGE_URL\@,$(PACKAGE_URL),g' \
-		-e 's,@RANDOM_SEED\@,$(localstatedir)/lib/random-seed,g' \
+		-e 's,@RANDOM_SEED\@,$(localstatedir)/$(rootlibdir)/random-seed,g' \
 		-e 's,@prefix\@,$(prefix),g' \
 		-e 's,@exec_prefix\@,$(exec_prefix),g' \
 		-e 's,@libdir\@,$(libdir),g' \
@@ -3619,9 +3619,9 @@
 	$(MKDIR_P) -m 0755 \
 		$(DESTDIR)$(tmpfilesdir) \
 		$(DESTDIR)$(sysconfdir)/tmpfiles.d \
-		$(DESTDIR)$(prefix)/lib/modules-load.d \
+		$(DESTDIR)$(prefix)/$(rootlibdir)/modules-load.d \
 		$(DESTDIR)$(sysconfdir)/modules-load.d \
-		$(DESTDIR)$(prefix)/lib/sysctl.d \
+		$(DESTDIR)$(prefix)/$(rootlibdir)/sysctl.d \
 		$(DESTDIR)$(sysconfdir)/sysctl.d \
 		$(DESTDIR)$(systemshutdowndir) \
 		$(DESTDIR)$(systemsleepdir) \
